src/Microsoft.AspNet.Mvc.Core/ActionResults/CreatedAtActionResult.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.AspNet.Mvc.Core;
+using Microsoft.AspNet.WebUtilities;
```

Comment:
Microsoft.aspnet.http ?

-----Original Message-----
From: "Chris R" <notifications@github.com>
Sent: ‎1/‎29/‎2015 5:07 PM
To: "aspnet/Mvc" <Mvc@noreply.github.com>
Cc: "Yishai Galatzer" <yishai_galatzer@yahoo.com>
Subject: Re: [Mvc] Replaced status code number with constants (#1924)

In src/Microsoft.AspNet.Mvc.Core/ActionResults/CreatedAtActionResult.cs:
> @@ -4,6 +4,7 @@
>  using System;
>  using System.Collections.Generic;
>  using Microsoft.AspNet.Mvc.Core;
> +using Microsoft.AspNet.WebUtilities;
With or without moving it to a new assembly?
Suggestions?
—
Reply to this email directly or view it on GitHub.=